### DISCUSSION TITLE
OTAPI= Reference?

### DISCUSSION THREAD

**Discussion by EdoanR** [Pryaxis/TShock#2082] [Q&A] [answered]
I have Visual Studio but having struggle to debug TShock. Mostly because I don't know how to find `OTAPI=` reference. I'm cloning the repository and opening the TShock.sln, references like `OTAPI` `TerrariaServer` I'm getting from the last TShock release, but `OTAPI=` I'm not sure how to find.

![Screenshot_1](https://user-images.githubusercontent.com/42870577/86152732-000fec80-bad7-11ea-9388-3ad35dade0bd.png)

**Comment by Quinci135** [comment, marked as answer]
OTAPI= is just looking for the tshock modified otapi that gets placed in a release or debug folder (depending on your build configuration) when you are patching OTAPI with tshock's own modifications. You can just delete the OTAPI= reference since you've already added one.
